### ISSUE TITLE
Bad primary-foreground contrast

### ISSUE THREAD

**Issue by curatedcode** [curatedcode/radix2shadcn#4]
**Describe the bug**
The contrast between primary and its foreground text is not good and needs to be improved. It has <4.5 contrast ratio sometimes.

**To Reproduce**

1. Go to '/'
2. Inspect contrast of default button with devtools
3. See contrast

**Expected behavior**
Accent foreground should be >=4.5 every single time.

**Screenshots**
A random initial palette generated
<img width="324" height="306" alt="Image" src="https://github.com/user-attachments/assets/fef31bbb-4cde-448c-87fb-e53275307e21" />

**Desktop (please complete the following information):**
All

**Smartphone (please complete the following information):**
All

**Additional context**
It's like a 80/20 chance to get a bad contrast against the primary, while not horrible it should be improved to 100% passing [wcag standards](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#rationale-for-the-ratios-chosen).
